FitOn/urls.py

```diff
@@ -14,52 +14,37 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from .views import (
-    confirm_deactivation,
-    deactivate_account,
-    delete_post_view,
-    fitness_trainer_application_view,
-    fitness_trainer_applications_list_view,
-    forum_view,
-    homepage,
-    login,
-    new_thread_view,
-    password_reset_complete,
-    password_reset_confirm,
-    password_reset_done,
-    password_reset_request,
-    profile_view,
-    signup,
-    thread_detail_view,
-    upload_profile_picture_view,
-)
+# from .views import signup, homepage, login, password_reset_complete, password_reset_confirm, password_reset_request, password_reset_done, profile_view, upload_profile_picture, deactivate_account, confirm_deactivation
+from . import views
 from django.conf import settings
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.contrib.auth import views as auth_views
 from django.urls import path
 
 urlpatterns = [
-    path('', login, name='landing'),
+    path('callback/', views.callback_google_fit, name='callback_google_fit'),
+    path('authorize/', views.authorize_google_fit, name='authorize_google_fit'),
+    path('', views.login, name='landing'),
     path('admin/', admin.site.urls),
-    path('signup/', signup, name='signup'),
-    path('home/', homepage, name='homepage'),
-    path('login/', login, name='login'),
-    path('deactivate/', deactivate_account, name='deactivate_account'),
-    path('deactivate/confirm/', confirm_deactivation, name='confirm_deactivation'),
-    path('logout/', login, name='logout'),
-    path('reset-password/', password_reset_request, name='password_reset_request'),
-    path('reset-password/<str:user_id>/<str:token>/', password_reset_confirm, name='password_reset_confirm'),
-    path('reset-password/done/', password_reset_done, name='password_reset_done'),
-    path('reset-password/complete/', password_reset_complete, name='password_reset_complete'),
-    path('profile/', profile_view, name='profile'),
-    path('upload_profile_picture/', upload_profile_picture_view, name='upload_profile_picture'),
```

Comment:
It should be "upload_profile_picture_view" like in the main branch

Reply:
resolved